enos/modules/build_local/main.tf

```diff
@@ -58,10 +58,12 @@ variable "vault_product_version" {
 }
 
 resource "enos_local_exec" "build" {
-  content = templatefile("${path.module}/templates/build.sh", {
-    bundle_path = var.bundle_path,
-    build_tags  = join(" ", var.build_tags)
-    goarch      = var.goarch
-    goos        = var.goos
-  })
+  content = "${path.module}/templates/build.sh"
```

Comment:
this can now be a script, since we are updating it to use environment variables instead of template